src/api/spec/features/webui/requests/submissions_spec.rb

```diff
@@ -105,9 +105,7 @@
     end
 
     describe 'when under the beta program' do
-      before { Flipper.enable(:request_show_redesign, receiver) }
-
-      describe 'submit several packages at once against a factory staging project' do
+      describe 'submit several packages at once against a factory staging project', beta: true do
```

Comment:
I think it makes more sense to add `beta: true` to the previous describe, which sets the "context" "_when under the beta program_".

Reply:
So true, changing